tests/wiTest.cpp

```diff
@@ -1768,11 +1780,21 @@ TEST_CASE("WindowsInternalTests::HandleWrappers", "[resource][unique_any]")
 #endif
 
     {
-        auto verify = MakeSecureDeleterMallocSpy();
-        REQUIRE_SUCCEEDED(::CoRegisterMallocSpy(verify.Get()));
-        auto removeSpy = wil::scope_exit([&] {
-            ::CoRevokeMallocSpy();
-        });
+        witest::detoured_thread_function<&::LocalFree> detour;
+        REQUIRE_SUCCEEDED(detour.reset([](HLOCAL p) -> HLOCAL {
+            HLOCAL h = nullptr;

```

Comment:
`wil::unique_hlocal`?

Reply:
Not sure that helps much here. `h` is the result of the call to `LocalFree` and is returned to the caller. We also couldn't use it for wrapping `p` since we need the result of the call to `LocalFree` to return to the caller.